instrumentation/jaxrs-client/jaxrs-client-2.0-testing/src/test/groovy/ResteasyProxyClientTest.groovy

```diff
@@ -11,12 +11,7 @@ import org.jboss.resteasy.specimpl.ResteasyUriBuilder
 import spock.lang.AutoCleanup
 import spock.lang.Shared
 
-import javax.ws.rs.GET
-import javax.ws.rs.HeaderParam
-import javax.ws.rs.POST
-import javax.ws.rs.PUT
-import javax.ws.rs.Path
-import javax.ws.rs.QueryParam
+import javax.ws.rs.*
```

Comment:
maybe revert changes to this file since https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/12903 converts it to java and there are only formatting changes